astropy/coordinates/tests/accuracy/test_altaz_icrs.py

```diff
@@ -55,7 +55,7 @@ def test_against_hor2eq():
     # radec_expected = ra=3.30875 deg, dec=15.183416666666666 deg
     # radec_actual = ra=3.3094193224314625 deg, dec=15.183757021354532 deg
     # distance = 2.6285 arcsec
-    assert distance < 5 * u.arcsec
+    assert distance < 10 * u.arcsec
```

Comment:
This was probably due to the iers-b issue.  I agree it should be possible to revert this change.